src/commons/stores/user-store.js

```diff
@@ -0,0 +1,46 @@
+(function(){
+	"use strict";
+
+	angular
+		.module("TendrlModule")
+		.service("userStore", userStore);
+
+	/*@ngInject*/
+	function userStore($state, $q, utils) {
+		var store = this;
+
+		store.getUserList = function() {
+			var list,
+				deffered;
+
+			deffered = $q.defer();
+			utils.getUserList()
+				.then(function(data) {
+					deffered.resolve(data);
+				});
+
+				return deffered.promise;
+		};
+
+
+		store.generateName = function(name) {
+				var userName = [],
+					temp1 = [],
+					temp2,
+					i,
+					len = name.length;
+
+				for (i = 0; i <len; i++) {
+					temp1 = name[i].split(" ");
+					temp2 = {};
+					temp2.firstName = temp1[0];
+					temp2.lastName = temp1[1];
+					userName.push(temp2)
+				}
```

Comment:
@gnehapk, we have to show First name and last name separately, and API will provide just the name, So I wrote this function to split name into 2 parameters. Since we will be sending (firstName+lastName) as name from add user form.